code/src/WijmoProvider/Features/ValidationMark.ts

```diff
@@ -157,16 +157,20 @@ namespace WijmoProvider.Feature {
             if (action.dataItem === undefined) return;
 
             const binding = this._grid.provider.getColumn(action.col).binding;
```

Comment:
This `const binding` can be defined inside the `if (typeof action._oldState !== 'object')`

Reply:
fixed